packages/stores/types.ts

```diff
@@ -846,6 +846,8 @@ type TUiStore = {
     setIsTradingDisabledByResidenceModal: (value: boolean) => void;
     should_show_same_dob_phone_modal: boolean;
     setShouldShowSameDOBPhoneModal: (value: boolean) => void;
+    field_ref_to_focus: string | null;
+    setFieldRefToFocus: (value: string | null) => void;
```

Comment:
these values are only for personal details form. isnt?
can we add personal_details_form_field to be specific ? or we can adda comment in the code for future reference

Reply:
Will add a comment. It is not specific to personal details